scalalib/src/mill/scalalib/scalafmt/ScalafmtModule.scala

```diff
@@ -0,0 +1,36 @@
+package mill.scalalib.scalafmt
+
+import ammonite.ops._
+import mill._
+import mill.define.{Command, Sources, Worker}
+import mill.scalalib._
+
+trait ScalafmtModule extends JavaModule {
+
+  def reformat(): Command[Unit] = T.command {
+    worker().reformat(
+      filesToFormat(sources()),
+      scalafmtConfig().head,
+      scalafmtDeps().map(_.path)
+    )
+  }
+
+  def scalafmtVersion: T[String] = "1.5.1"
+
+  def scalafmtConfig: Sources = T.sources(pwd / ".scalafmt.conf")
+
+  def scalafmtDeps = resolveDeps(
```

Comment:
Should we use certain scala version to resolve deps to stay stable no matter what version of scala mill uses?

Reply:
I don't have any opinion, though I suppose scalafmt is meant to do the same thing regardless of scalaVersion?

Reply:
Yes, version of scala doesn't affect the result of formatting with scalafmt. What I meant is the situation like this: 
At some point, scala 2.13 is released. We decide to migrate mill to 2.13, but we won't be able to do that until scalafmt for 2.13 is released.